book/src/release_management.md

```diff
@@ -29,7 +29,7 @@ It is also sufficient to tag only the most high-level crate being published.
 The crates must be published in the order below.
 
 - [Traits](https://github.com/openmls/openmls/blob/main/traits/Cargo.toml)
-- [Memory Keystore](https://github.com/openmls/openmls/blob/main/memory_keystore/Cargo.toml)
+- [Memory storage](https://github.com/openmls/openmls/blob/main/memory_storage/Cargo.toml)
```

Comment:
I didn't scan fully in the book for correct changes in the use of the terms yet, seems likely there are more incorrect references, not just broken links.